main/guides/js-programming/hardened-js.md

```diff
@@ -79,6 +79,20 @@ e.g., only giving the `entryGuard` the ability to increment the counter.
 
 This limits the damage that can happen if there is an exploitable bug.
 
+### Widely Shared vs. Closely Held
+
+#### Widely Shared Resources
+
+In the context of Agoric's object capability model, "widely shared" refers to resources or capabilities that are accessible to a large portion of the code within a system. For example:
```

Comment:
This makes it sound like the term is not used outside of Agoric. It dates back at least as far as [Keykos](https://en.wikipedia.org/wiki/KeyKOS) in 1977

http://www.cap-lore.com/CapTheory/Glossary.html

Reply:
Wow. Thanks for sharing this!